recipes/vectorclass/all/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.8)
```

Reply:
to use `cxx_std_17` compile feature (`CMAKE_CXX_STANDARD 17` requires CMake 3.8 anyway)

Reply:
are there some guidelines what minimum CMake version should be supported in CCI?